src/cmd/build.js

```diff
@@ -56,8 +90,17 @@ async function defaultPackageCreator(
     filter: (...args) => fileFilter.wantFile(...args),
   });
 
+  let messageData: any;
+  if (manifestData.default_locale) {
+    messageData = await fs.readFile(path.join(sourceDir, '_locales',
```

Comment:
The `readFile()` function needs to move into `getDefaultLocalizedName()` because when you throw an error about malformed JSON, the error should include the filename. This will help the developer track down and fix the JSON syntax error more easily.

At first I thought we could avoid it but it needs to include the filename. Unfortunately, this means testing for syntax errors is a bit harder. You will have to pass it a real file instead of a string of JSON. You can use `withTempDir` to write a temporary file for the test. It would look something like:

````javascript
it('checks locale file for malformed json', () => {
  return withTempDir(
    (tmpDir) => {
      const messageFile = path.join(tmpDir, 'messages.json');
      return fs.writeFile(messageFile, '{"simulated:" "json syntax error"'))
        .then(() => {
          // assertion code
        });
    }
  );
});
````